src/main/java/de/tum/in/www1/artemis/service/ProgrammingExerciseExportService.java

```diff
@@ -92,15 +101,15 @@ public File exportStudentRepositories(long programmingExerciseId, @NotNull List<
                     + "' of the following students or teams: " + participations.stream().map(StudentParticipation::getParticipantIdentifier).collect(Collectors.joining(", ")));
         }
 
-        List<Path> zippedRepoFiles = Collections.synchronizedList(new ArrayList<>());
+        List<Path> pathsToZippedRepoFiles = Collections.synchronizedList(new ArrayList<>());
```

Comment:
Codacy found an issue: [Local variable 'pathsToZippedRepoFiles' could be declared final](https://app.codacy.com/gh/ls1intum/Artemis/pullRequest?prid=5968852)